code/modules/bitrunning/components/service_style.dm

```diff
@@ -0,0 +1,30 @@
+/// Gives the appearance of being an agent
+/datum/element/service_style
```

Comment:
this is kinda a crummy element imo, it doesn't really do anything event based it's basically just a fancy proc on human. pagin ninja he'd know better

Reply:
it's entirely just a fancy proc on human, as are the others. however with this solution it just lets admins use addelement if they want

Reply:
Admins can also just call the proc though